site/docs/master/start-contributing.md

```diff
@@ -19,3 +21,5 @@ If you are ready to jump in and test, add code, or help with documentation, plea
 
 [1]: https://github.com/vmware-tanzu/velero/blob/master/CODE_OF_CONDUCT.md
 [2]: https://github.com/vmware-tanzu/velero/blob/master/CONTRIBUTING.md
+[3]: https://github.com/vmware-tanzu/velero/blob/master/.github/ISSUE_TEMPLATE/feature-enhancement-request.md
```

Comment:
per prev comment I'd duplicate this change in the v1.4 docs as well, but you'll want to point the links to the v1.4.0 branch (just look at the other links in the v1.4 version of the file)

Reply:
I can do that, but then I'd have to change the issue templates in the v1.4.0 branch as well, correct?

Reply:
Ah, you're right. I guess in this case leaving them pointed to `master` makes sense. Next time we generate a new versioned docs site, this'll all work as expected.

Reply:
Perfect!